lab_2_CSS.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+
+<meta charset="UTF-8"/>
+
+<title>Lab2_1</title>
+
+<link href="lab2_style.css" rel="stylesheet">
+
+</head>
+
+<body>
+
+<h1>Селекторы</h1>
+
+<p>Поверхность Мохо несовершенна. Сталагмит <em>зеркально</em> отражает кризис. Аутизм притягивает гомеостаз. Распространиение вулканов осознаёт экспериментальный ортоклаз. Наш современник стал особенно чутко относиться к слову, однако психоанализ значительно иллюстрирует интеграл Гамильтона, как и предполагалось. Денудация <em>положительна</em>.</p>
+
+<p>Механизм власти фактически означает механизм власти. <em>Очевидно</em>, что упаривание теоретически катализирует полимолекулярный ассоциат. Мицелла лимитированно возбуждает диэтиловый эфир. Политическая психология гомологична. Впервые газовые гидраты были описаны Гемфри Дэви в 1810 году, <em>однако</em> унитарное государство ядовито.</p>
+
+<div id="allotment" class="plants-list">
+
+<h2>Огород</h2>
+
+<ul>
+
+<li>Фрукты:
+
+<ol class="fruits-list">
+
+<li>Апельсин</li>
+
+<li>Слива</li>
+
+<li>Яблоко</li>
+
+</ol>
+
+</li>
+
+<li>Овощи:
+
+<ol class="vegetables-list">
+
+<li>Помидор</li>
+
+<li>Картошка</li>
+
+<li>Огурец</li>
+
+</ol>
+
+</li>
+
+</ul>
+
+</div>
+
+<div id="warehouse" class="plants-list">
+
+<h2>Склад</h2>
+
+<ul>
+
+<li>Фрукты:
+
+<ol class="fruits-list">
+
+<li>Вишня</li>
+
+<li>Груша</li>
+
+<li>Яблоко</li>
+
+</ol>
+
+</li>
+
+<li>Овощи:
+
+<ol class="vegetables-list">
+
+<li>Картошка</li>
+
+<li>Баклажан</li>
+
+</ol>
+
+</li>
+
+</ul>
+
+</div>
+
+<table class="students-table" width="400" border="1">
```

Comment:
The border attribute of <table> is not supported in HTML5. Use CSS instead.
https://www.w3schools.com/TagS/att_table_border.asp